src/styles/index.scss

```diff
@@ -1,3 +1,145 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  background: grey;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  background: #fff;
+  width: 198px;
+  height: 406px;
+  top: 1px;
+  left: 1px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card--image {
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: 160px 134px;
+  background-position: 19px 32px;
+}
+
+.card__text1 {
+  font-weight: 500;
+  font-size: 12px;
+  width: 166px;
+  height: 36px;
+  margin-top: 206px;
+  margin-left: 16px;
+}
+
+.card__text2 {
+  color: #616070;
+  font-weight: 400;
+  font-size: 10px;
+  width: 98px;
+  height: 14px;
+  margin-top: -8px;
+  margin-left: 17px;
+}
+
+ul,
+li {
+  list-style: none;
+}
+
+.content {
+  display: flex;
+  justify-content: space-between;
+}
+
+.content__stars {
+  display: flex;
+  margin-top: 6px;
+  justify-content: space-between;
+  margin-left: -23px;
+}
+
+.stars {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.content__stars .stars:last-child {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.content__review {
+  color: #060b35;
+  font-size: 10px;
+  font-weight: 400;
+  width: 53px;
+  height: 16px;
+  line-height: 14px;
+  margin-right: 11px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: -8px;
+}
+
+.text__3 {
+  color: #616070;
+  width: 31px;
+  height: 18px;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  margin-left: 16px;
+}
+
+.price__text {
+  color: #000;
+  height: 18px;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 10px;
+  margin-right: 16px;
+}
+
+.buttom {
+  background-color: #00acdc;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin-left: 16px;
+  margin-bottom: 16px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.button__text {
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  padding-top: 12px;
+}
+
+.buttom:hover {
```

Comment:
You can also use _Nesting_ for `&:hover`, so:

`.button:hover`
is the same as
```
.button {
  &:hover {
 }
}
```